Nim/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿Console.WriteLine("Welcome to Nim");
+Console.WriteLine("Rules are: You may only draw up to 3 matches.");
+
+int matches = 24; //Starting matches
+
+Console.WriteLine("|||||||||||||||||||||||| (24)"); //Presenting total matches
+Console.WriteLine("Player 1: How many matches do you want to draw?");
+
+round:
+tryAgain:
```

Comment:
I like that you used two separate labels, even though they point at the same line of code. This is a great example of "Just because two things LOOK the same doesn't mean they ARE the same. Maybe, you want to move one of these labels at a later point. Starting a new round and having the player attempt making a valid input again are two separate things.